fthd_v4l2.c

```diff
@@ -41,6 +41,12 @@
 #define FTHD_MIN_HEIGHT 240
 #define FTHD_NUM_FORMATS 2 /* NV16 is disabled for now */
 
+/* This trick taken from
+ * https://github.com/umlaeute/v4l2loopback/commit/4815caf32e5dceb0a0cad02d9522d0a44d1defd6 */
```

Comment:
This information is better to keep in the git commit itself. No need to clutter the code with it.

Reply:
Ok, I updated the PR to remove those comments and put them in the commit message